Sources/RCam/Core/CameraImpl.swift

```diff
@@ -90,7 +93,17 @@ public final class CameraImpl: Camera {
         }
     }
 
-    public var zoomRangeLimits: ClosedRange<CGFloat>? = 1...5
+    public var zoomRangeLimits: ClosedRange<CGFloat>? {
+        if usingBackCamera {
+            return Constants.backCameraZoomRange
+        }
+        else {
+            return Constants.frontCameraZoomRange
+        }
+    }
+    public var frontCameraZoomFactor: CGFloat {
```

Comment:
can we zoom this property only in this service and use `zoomRangeLimits` how it is, but correct it in `func update(zoomLevel: CGFloat)` method in line 309